pkg/upstream/transformer.go

```diff
@@ -162,9 +162,23 @@ func (t *StaticRedirectTransform) Modify(r *Request) (*Request, error) {
 // - contrib.go.opencensus.io
 // - go.uber.org
 type GoGetTransform struct {
-	domains    map[string]bool // only implement redirect metadata
-	httpClient *http.Client
-	log        loggy.Logger
+	redirectAll bool
```

Comment:
`redirectAll` isn't quite semantically correct - not all queries get redirected, only the ones that respond to `?go-get=1` - everything else should just use the initial name without redirecting (e.g. everything using github.com).
maybe call it `autoRedirect` ?